azure-pipelines-official.yml

```diff
@@ -14,35 +14,56 @@ jobs:
 
   variables:
     BuildPlatform: 'Any CPU'
-    InsertTargetBranchFullName: 'lab/d16.0stg'
-    InsertTargetBranchShortName: 'd16.0stg'
+    VisualStudio.MajorVersion: 16
+    VisualStudio.BranchName: 'lab/d16.0stg'
+    VisualStudio.ChannelName: 'int.d16.0stg'
+    VisualStudio.DropName: Products/$(System.TeamProject)/$(Build.Repository.Name)/$(Build.SourceBranchName)/$(Build.BuildNumber)
+
   steps:
+  - task: NuGetToolInstaller@0
+    inputs:
+      versionSpec: '4.9.2' 
+
   - task: NuGetCommand@2
+    displayName: 'Clear NuGet caches'
     inputs:
       command: custom
       arguments: 'locals all -clear'
 
+  - task: NuGetCommand@2
+    displayName: Restore internal tools
+    inputs:
+      command: restore
+      feedsToUse: config
+      restoreSolution: 'eng\internal\Toolset.csproj'
+      nugetConfigPath: 'NuGet.config'
+      restoreDirectory: '$(UserProfile)\.nuget\packages'
+
   - task: ms-vseng.MicroBuildTasks.30666190-6959-11e5-9f96-f56098202fef.MicroBuildSigningPlugin@1
     inputs:
       signType: real
+      zipSources: false
```

Comment:
Why did we make this change?

Reply:
Because we don't need the zipped sources. We used to push the zip file with sources to cpvsbuild. Now we don't do anything with that file, so there seems to be no point in creating it.

Reply:
(I just noticed while watching the build that it's zipping the sources).

Reply:
The zip file though ends up in the published artifacts. Did last time Ichecked. Is this just not putting that in the artifacts anymore? If so I agree taht's fine.

Reply:
Ah, you are right, it's in Microbuild outputs. I think that was rather accidental, since we started publishing Microbuild outputs only because we needed the BootstrapperInfo.json file from there in this change: https://github.com/dotnet/roslyn/commit/a91e9f565bbaf3ffa386484561f7d4af75bb689d#diff-f54ede2a43d00dde2f6ff27364db0e9dR116

I can turn this on again if we think it's valuable to archive the sources.